ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.cs

```diff
@@ -7,6 +7,7 @@
 using DesktopUI2.Models.Filters;
 using DesktopUI2.Models.Settings;
 using DesktopUI2.ViewModels;
+using Newtonsoft.Json;
```

Comment:
We should not use `Newtonsoft.Json` as it might conflict, should instead use our fork `Speckle.Newtonsoft.Json`

Reply:
Ah good catch, swapped this out 👍